cabin-sorting/data/csv_inputs/answer.csv

```diff
@@ -1,7 +1,6 @@
-question0, question1, question2, question3, question4, question5, question6, question7, question8, question9, question10, question11
-answer1, answer1, answer1, answer1, answer1, answer1, answer1, answer1, answer1, answer1, answer1, answer1
-answer2, answer2, answer2, answer2, answer2, answer2, answer2, answer2, answer2, answer2, answer2, answer2
-answer3, answer3, answer3, answer3, answer3, answer3, answer3, answer3, answer3, answer3, answer3, answer3
-answer4, answer4, answer4, answer4, answer4, answer4, answer4, answer4, answer4, answer4, answer4, answer4
-answer5, answer5, answer5, answer5, answer5, answer5, answer5, answer5, answer5, answer5, answer5, answer5
-answer6, answer6, answer6, answer6, answer6, answer6, answer6, answer6, answer6, answer6, answer6, answer6
+question0|question1|question2|question3|question4|question5|question6|question7|question8
```

Comment:
we will receive our data in the form of actual question text being in the header, not "question0", "question1" etc. i saw that this format is part of the algorithm implementation - please change that

Reply:
I see the standardization you guys are trying to accomplish! Yes, change for now! At a later point, we can standardize.